__mocks__/wardBeds.mock.ts

```diff
@@ -0,0 +1,31 @@
+import { mockBedType } from './wards.mock';
+
+export const mockWardBeds = [
```

Comment:
How does this relate to the existing `mockAdmissionLocation` mock in the wards.mock file?  Can we just use that other existing mock?

Reply:
Couple of properties such as id and uuid are named differently in `mockAdmissionLocation `for beds with some additional properties,so i just created a separate mock for beds.